rumqttd/src/link/bridge.rs

```diff
@@ -198,62 +198,50 @@ pub async fn tls_connect<P: AsRef<Path>>(
 ) -> Result<Box<dyn N>, BridgeError> {
     let mut root_cert_store = RootCertStore::empty();
 
-    let ca_certs = rustls_pemfile::certs(&mut BufReader::new(Cursor::new(fs::read(ca_file)?)))?;
-    let trust_anchors = ca_certs.iter().map_while(|cert| {
-        if let Ok(ta) = webpki::TrustAnchor::try_from_cert_der(&cert[..]) {
-            Some(OwnedTrustAnchor::from_subject_spki_name_constraints(
-                ta.subject,
-                ta.spki,
-                ta.name_constraints,
-            ))
-        } else {
-            None
-        }
-    });
-
-    root_cert_store.add_trust_anchors(trust_anchors);
+    let ca_certs = rustls_pemfile::certs(&mut BufReader::new(Cursor::new(fs::read(ca_file)?)))
+        .collect::<Result<Vec<_>, _>>()?;
+    root_cert_store.add_parsable_certificates(ca_certs);
```

Comment:
also using `add_parsable_certificates` might leave out some certs right?

wdyt of rewriting this section ( from line 201-103 ) as:
```rust
for cert in rustls_pemfile::certs(&mut BufReader::new(Cursor::new(fs::read(ca_file)?))) {
    root_cert_store.add(cert?)?
}
```